### DISCUSSION TITLE
Calculator Master - zCalc and OpenCalc duplicate or breaking of the license

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#397] [General]
https://apps.microsoft.com/detail/XPFG5JJ4WSXVW1?hl=en-US&gl=US

I didnt find the source code.

It looks like phone app, but its on  [Microsoft Apps Store](https://apps.microsoft.com/home?hl=en-US&gl=US), which is for desktop apps, right?

I didnt try it, it costs money + It needs WIN 11 (Wtf?) and I have WIN 10.

**Comment by clementwzk** [comment]
lol, thanks for your report, I will try to remove that shit this weekend

**Reply by inson1** [reply]
@Darkempire78 well, if I can help with that somehow, say it :)

**Comment by inson1** [comment]
Its even on Google play store https://play.google.com/store/apps/details?id=com.fla.calc

Even THEME bug is there xD

**Reply by inson1** [reply]
"Owner" answered me 
"Hello, of course I can remove it. Could you please let me know how it violates the Opencalc license?

If I add the source code will the problem be solved?"

**Reply by inson1** [reply]
"Hello, of course I can remove it. Could you please let me know how it violates the Opencalc license? If I add the source code will the problem be solved? Can you reply and I will remove it accordingly? I will direct you to the opencalc page of Darkempire78, who shares the application as source code on github."

**Reply by clementwzk** [reply]
@inson1 He needs to disclose the source, to use the same license, mention OpenCalc (copyright)
To be honest, I'd prefer them to delete the application and, above all, not to charge for something that's identical for free.

**Reply by inson1** [reply]
@Darkempire78 and then there is also google play policy about duplicates

**Comment by clementwzk** [comment]
He is respecting the license now , all is good!
Thanks @inson1
